rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -41,7 +51,45 @@ impl Tun {
             }
         };
 
-        adapter.set_address(config.ipv4)?;
+        // TODO: I think wintun flashes a couple console windows here when it shells out to netsh. We should upstream the same patch I'm doing for powershell to the wintun project
+        adapter.set_network_addresses_tuple(
```

Comment:
For the future, I'm not a fan of depending on the `wintun` crate for this(or even using it at all tbh) we should reimplement them using win32 api(save for setting ipv6 address which can be only done shelling out)